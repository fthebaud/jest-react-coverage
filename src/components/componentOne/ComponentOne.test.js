import React from 'react';
import { create } from "react-test-renderer";
import ComponentOne from './ComponentOne';

describe('ComponentOne', () => {
    test('Snapshot testing', () => {
        const component = create(<ComponentOne />);
        expect(component.toJSON()).toMatchSnapshot();
    })
})