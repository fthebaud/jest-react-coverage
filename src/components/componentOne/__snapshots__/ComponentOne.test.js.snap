// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentOne Snapshot testing 1`] = `
<div>
  COMPONENT ONE
</div>
`;
